src/ck-core/ckarray.h

```diff
@@ -87,191 +88,242 @@ class CkVerboseListener : public CkArrayListener {
 /*@{*/
 
 /*********************** Proxy Support ************************/
-//Needed by CBase_ArrayElement
-class ArrayBase { /*empty*/ };
+// Needed by CBase_ArrayElement
+class ArrayBase
+{ /*empty*/
```

Comment:
Would it be better to simply delete `/*empty*/` under the new formatting guidelines?

Reply:
Done